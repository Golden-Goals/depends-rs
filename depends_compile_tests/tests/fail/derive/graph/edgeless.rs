use depends::derives::Graph;
#[allow(unused_imports)]
use examples::*;

#[derive(Graph)]
#[depends(
    digraph Dag {
        comments [label="Comments"];
        likes [label="Likes"];
    }
)]
struct Graph {}

fn main() {}
