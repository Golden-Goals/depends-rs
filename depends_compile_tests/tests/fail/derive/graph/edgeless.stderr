error: A Graphviz model must have at least 1 edge.
 --> tests/fail/derive/graph/edgeless.rs:5:10
  |
5 | #[derive(Graph)]
  |          ^^^^^
  |
  = note: this error originates in the derive macro `Graph` (in Nightly builds, run with -Z macro-backtrace for more info)
